src/sqlfluff/dialects/dialect_sparksql.py

```diff
@@ -346,6 +346,26 @@
         # Add arrow operators for lambdas (e.g. aggregate)
         Ref("RightArrowOperator"),
     ),
+    # Support for colon sign operator (Databricks SQL)
+    ObjectReferenceDelimiterGrammar=OneOf(
+        Ref("DotSegment"),
+        Sequence(Ref("DotSegment"), Ref("DotSegment")),
+        Ref("ColonSegment"),
+    ),
```

Comment:
could it be `OneOf` the `ansi.ObjectReferenceDelimiterGrammar` (instead of repeating the first two) or `Ref("ColonSegment")`?

Reply:
I did not change it, just moved it. I need to take a look at it but it has to be refactored for Databricks in anyway. I have a note and a plan for it. Right now, SQLFluff fails to parse [Databricks SQL JSON path expression](https://docs.databricks.com/sql/language-manual/sql-ref-json-path-expression.html).

As it is kept previous state, I don't think it's a blocker. @greg-finley

Reply:
OK, makes sense. Thanks!